packages/common/src/ui/layout/tables/context/TableContext.ts

```diff
@@ -17,6 +18,7 @@ export interface TableStore {
   toggleSelected: (id: string) => void;
   toggleAll: () => void;
   setActiveRows: (id: string[]) => void;
+  setDisabledRows: (id: string[]) => void;
```

Comment:
I added this mainly because I wanted to check that setting a row as disabled wouldn't be too difficult (i.e. if we wanted to use it for `ItemDetailsModal`)